service/api/types.go

```diff
@@ -295,6 +295,8 @@ type DebuggerCommand struct {
 	ReturnInfoLoadConfig *LoadConfig
 	// Expr is the expression argument for a Call command
 	Expr string `json:"expr,omitempty"`
+	// If UnsafeCall is set the function call will not be checked for escapes
```

Comment:
nit: start doc string with name of exported symbol, e.g. "UnsafeCall disabled parameter escape checking for function calls".